packages/collector/src/logger.js

```diff
@@ -99,6 +109,11 @@ exports.getLogger = function getLogger(loggerName, reInitFn) {
     _logger = parentLogger;
   }
 
+  // Extended method to set log level
+  _logger.setLoggerLevel = function (/** @type {string | number} */ level) {
```

Comment:
IMO this is a wrong semantical place. Define the fn once in the init fn. Not here please